### ISSUE TITLE
PowerNorm

### ISSUE THREAD

**Issue by deJQK** [pytorch/pytorch#55274]
## 🚀 Feature
The PowerNorm operation proposed in [ICML2020](https://arxiv.org/pdf/2003.07845.pdf).

## Motivation

Is it possible to add the PowerNorm operation from [this repo](https://github.com/sIncerass/powernorm), as it is reported to relax the zero mean requirement of batch norm and reduce the mean variation between training and evaluation of the model, which in turn improves the final performance of the model.
